ci/tests_all_in_schedule.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -euo pipefail
+
+# shellcheck disable=SC1091
+source ci/ci_helpers.sh
+cd src/test/regress
+
+for test_dir in sql spec; do
+    test_names=$(find $test_dir -iname '*.sql' | sed "s#^$test_dir/\\(.*\\)\\.\\(sql\\|spec\\)\$#\\1#g")
+    for name in $test_names; do
+        if ! grep "\\b$name\\b" ./*_schedule > /dev/null; then
```

Comment:
could we simply use the quiet(--quiet|--silent) option of grep instead of redirecting to `/dev/null`?

Reply:
For portability, we should keep `/dev/null` here. GNU Grep manpage has a section on this.
> Portable  shell  scripts should  avoid  both  -q  and -s and should redirect standard and error output to /dev/null instead.  (-s is specified by POSIX.)

Reply:
Undid my initial change to `--silent`, because of what @hanefi said.